### PR TITLE
EE-1760: increased the minimum required Python version to 3.6

### PR DESCRIPTION
**Pull request by bela-juhasz**
Please see my discussion-starting comment here: https://www.ebi.ac.uk/panda/jira/browse/EE-1760?focusedCommentId=347911&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-347911

### CODE REVIEW

**Review by ashutoshebi — APPROVED** [review]

**Review by malloryfreeberg — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The pyEGA3 download client is a python-based tool for viewing and downloading fi
 
 ## Requirements
 
-* Python3 ([download instructions](https://www.python.org/downloads/))
+* Python 3.6 or newer. ([download instructions](https://www.python.org/downloads/))
```
Comment (bela-juhasz) [inline README.md:18]: Please see my discussion-starting comment here: https://www.ebi.ac.uk/panda/jira/browse/EE-1760?focusedCommentId=347911&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-347911

**Review by afoix — APPROVED** [review]
LGTM!

**Comment by malloryfreeberg** [comment]
Fixes #97 👍

**Pull request merged**